NhanVienKinhDoanh.dart

```diff
@@ -12,6 +12,12 @@ class NhanVienKinhDoanh extends NhanVien {
     }
   @override
   String toString() {
-    return super.toString() + '\t$doanhSo';
+    return super.toString() + '\t\t$doanhSo\t\t\t\t$realSalary';
   }
+  @override
+  void calSalary(){
```

Comment:
Code convention
Suggestion: from ){ to ) {

Reply:
I reformatted code with dart format.